spec/factories/edition_factory.rb

```diff
@@ -119,6 +119,25 @@
       end
     end
 
+    trait :removed do
+      live { true }
+
+      transient do
+        removal { nil }
+      end
+
+      after(:build) do |edition, evaluator|
+        edition.revision = evaluator.association(:revision)
```

Comment:
I don't think we should or need to do this, it seems to be a mistake we have with withdrawn factory too.